src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -179,12 +182,158 @@ namespace OpenLoco::Vehicles
         calculateRefundCost();
     }
 
+    // 0x004B8340
+    static void RecalculateTrainMinReliability(VehicleHead& head)
```

Comment:
Minor code style violation:
```suggestion
    static void recalculateTrainMinReliability(VehicleHead& head)
```